dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -157,6 +157,7 @@ class CDefaultContextMenu :
         UINT m_cKeys;
         PIDLIST_ABSOLUTE m_pidlFolder;
         DWORD m_bGroupPolicyActive;
+        UINT m_iIdQCMFirst;
```

Comment:
Suggestion: add a description comment like what's done for the other m_iId...First members below. It's not clear what "QCM..." means here.